### DISCUSSION TITLE
iPhone scrolling bug

### DISCUSSION THREAD

**Discussion by 0nelight** [community/community#39654] [Mobile]
Sometimes it happens that I scroll with iPhone, but the scrolling start to hang at a particular place and I cannot scroll beyond that.

See this Screencast for details:
https://drive.google.com/file/d/1MbGZFCARE_5c_z3s41U2jiSzo4vtPdFF/view?usp=drivesdk

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.89.0 (83472182)
Device: iPhone 6S (iOS 15.7.1)
</details>

**Comment by taki-on** [comment]
Thank you for reporting the issue. I believe it happens not always but sometimes, right? If so, the issue is known to us and we will look into it soon. Still, we are sorry for the inconvenience, and please stay tuned!
